xdmod/README.md

```diff
@@ -62,39 +68,218 @@ The following information is needed by Open XDMoD:
 Optionally:
 
 - An image file containing the HPC center logo
+    - The width HPC center logo
 
 Also the following technical information:
 
 - The public url of Open XDMoD
 - Paths to installed dependencies (phantomjs)
 - MySQL connection information
+    - Host
+    - Port
+    - Admin Username
+    - Admin Password
+    - DB Username
+    - DB Password
+
+### Prerequisites used in this Tutorial
+
+- Name of the organization: `Tutorial` abbreviation: `hpcts`
+- information for each HPC resource
+    - Name: `hpc`
+    - Number of compute nodes: `2`
+    - Number of cores: `2`
+    - Timezone: `UTC`
+    - Whether it runs shared jobs: `no`
+- An image file containing the HPC center logo: `/srv/xdmod/small-logo.png`
+    - The width HPC center logo: `354`
+- The public url of Open XDMoD:  `https://localhost:4443`
+- Paths to installed dependencies (phantomjs):  `detected defaults`
+- MySQL connection information
+    - Host: `mysql`
+    - Port: `3306`
+    - Admin Username: `root`
+    - Admin Password: ` leave blank `
+    - DB Username: `xdmodapp`
+    - DB Password: `ofbatgorWep0`
+
+### Basic Configuration
+Open XDMoD provides an interactive configuration script that performs the
+database initialization and generates configuration files. This script
+handles the basic setup.
+
+The [`hpc-toolset-tutorial/xdmod/entrypoint.sh`](https://github.com/ubccr/hpc-toolset-tutorial/blob/master/xdmod/entrypoint.sh) script automates this process.
+
+Reference: [Configuration Guide](https://open.xdmod.org/configuration.html)
+
+The following asciinema recordings are how an administrator would do perform these actions:
+
+General Setup:
+[![asciicast](https://asciinema.org/a/349236.svg)](https://asciinema.org/a/349236)
+
+Database Setup:
+[![asciicast](https://asciinema.org/a/349237.svg)](https://asciinema.org/349237)
+
+Organization Setup:
+[![asciicast](https://asciinema.org/a/349238.svg)](https://asciinema.org/349238)
+
+Resource Setup:
+[![asciicast](https://asciinema.org/a/349240.svg)](https://asciinema.org/349240)
+
+#### What about a Heterogeneous cluster?
+
+The `xdmod-setup` script is used for the basic setup of Open XDMoD. The script includes options to configure the Open XDMoD database, setup the admin user account and configure resources.
+Open XDMoD's [Configuration](https://open.xdmod.org/configuration.html#location-of-configuration-files) files.
+
+Resource Specs Modification:
+[![asciicast](https://asciinema.org/a/349249.svg)](https://asciinema.org/349249)
+
+
+#### Hierarchy
+
+Open XDMoD supports a three level hierarchy.
+In this tutorial we use a University Type Hierarchy.
+
+Decanal Unit -> Department -> PI Group
+
+Reference: [Hierarchy Guide](https://open.xdmod.org/hierarchy.html)
+
+#### User / PI Names
+
+Open XDMoD has the ability to import a full names.  If full names are not imported, usernames are used.
```

Comment:
The resource manager logs contain the system username of the user that submitted each job. To display the full names in Open XDMoD you must provide a data file that contains the full name of each user for each system username. This file is in a `csv` format.